Backend/ServerManager.py

```diff
@@ -204,12 +204,24 @@ def api_get_trends_snapshot():
     trends = request.args.get("trends")
     fromdate = request.args.get("from")     #YYYY-mm-dd HH:MM:SS
     todate = request.args.get("to")         #YYYY-mm-dd HH:MM:SS
+    loc = request.args.get("woeid")         #optional woeid argument
+
+    # args for the time step of each datapoint
+    day = request.args.get("days")
+    hour = request.args.get("hours")
+    min = request.args.get("minutes")
+    sec = request.args.get("seconds")
 
     try:
         if trends is not None:
             trendsparsed = trends.split(",")
             until = datetime.now()
             since = until - timedelta(hours=12)
+            woeid = 1
```

Comment:
I still don't understand when we would want this default value to exist. Why would we want to fetch all of the snapshots of a trend from all locations at once?